app/views/layouts/user_orders.html.erb

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```

Comment:
where is this layout used?

Reply:
This is used in extension/views branch
Will remove it from this branch